lib/minitest/assertions.rb

```diff
@@ -50,27 +50,32 @@ def self.diff= o
       @diff = o
     end
 
+    ##
+    # Returns things to diff [expect, butwas], or [nil, nil] if nothing to diff.
+    def things_to_diff(exp, act)
```

Comment:
no parens to match the project's style

Reply:
I think I've changed the things you requested.  Thanks, zenspider, for being patient.